### ISSUE TITLE
Want to know about if any API exist to find graphic instancing in ifc file

### ISSUE THREAD

**Issue by SuryaHexagonPPM** [xBimTeam/XbimEssentials#348]
Want to know about if any API exist to find graphic instancing in ifc file

#### Problem

We are using XBim lib to read ifc file in our product. I was wondering if there exist any API which will tell if a representation item is used by more than one IFCObject. E.g. In a IFC dataset, two IFCBeams can point to same IFCFACETEDBREP. If we get a list of entity labels (of representation item) which is used by more than one ifcelement it will help us to reduce the number of times we process same representation item.

**Comment by martin1cerny** [comment]
Yes, just use xbim toolkit to browse the data from product to shapes and compare their references.

**Comment by andyward** [comment]
Can we close this?
